src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -137,7 +139,9 @@ private AccordService()
         this.messageSink = new AccordMessageSink();
         this.configService = new AccordConfigurationService(localId);
         this.scheduler = new AccordScheduler();
+        this.config = new AccordConfig(DatabaseDescriptor.getAccordSchedulerDelayInMS());
```

Comment:
this pattern isn't something we can maintain; every new config comes with a massive burden to update this constructor; IMO we should push this logic into the YAML layer